docs/sources/administration/service-accounts/create-service-account.md

```diff
@@ -0,0 +1,35 @@
+---
+title: Create a service account in Grafana
+menuTitle: Create a service account
+aliases: [docs/sources/administration/service-accounts/add-service-account-token.md]
+description: 'How to create a service account in Grafana'
+weight: 50
+keywords:
+  - Service accounts
+---
+
+# Create a service account in Grafana
+
+You can create a service account in the service account tab of the organization. To manage your computed workload with a user that is not tied to a user specifically.
+
+Consider the following restriction when you create a service account:
+
+- A service account is identified by its name, which is unique to the entire suite of organizations. The name is set upon creation of the service account
+- Service accounts live on a organizational level and are restricted as such
```

Comment:
Find another word for 'live'

Reply:
What about "Service accounts are mapped to an organization where they are created"?